tests/fixtures.py

```diff
@@ -104,6 +104,21 @@ def setUp(self):
         build_files(DistInfoPkg.files, self.site_dir)
 
 
+class DistInfoPkgWithDot(OnSysPath, SiteDir):
+    files = {
+        "pkg.dot-1.0.0.dist-info": {
```

Comment:
Since `pkg_dot` is the proposed format, I think that should be the fixture. Perhaps this fixture should also exist as a "legacy" format.

Reply:
A fixture for this legacy format is definitely needed, since there are many `.dist-info` directories using it in the wild.

Reply:
Confirmed and added in e7d59299d5250c286b038a1131840d76d2007e95.